plugin/Installer/DatabaseChanges.php

```diff
@@ -781,4 +782,14 @@ public function setDetailTemplate()
 		$pDetailView->setTemplate( key( $firstTemplatePath ) );
 		$pDataDetailViewHandler->saveDetailView( $pDetailView );
 	}
+
+	public function updateShowReferenceEstateOfReferenceList()
+	{
+		$prefix = $this->getPrefix();
+		$sql = "UPDATE {$prefix}oo_plugin_listviews
+				SET `show_reference_estate` = 1
+				WHERE `list_type` = 'reference'";
```

Comment:
This was a bit ambiguous in the requirements, but this should be set for all list types.

The reason is that previously, reference estates were always shown, for all list types. So for backwards compatibility, we need to keep showing reference estates.